packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -444,7 +445,7 @@ module.exports = async ({
   plugins.minifyJs = ({ terserOptions, ...options } = {}) =>
     new TerserPlugin({
       cache: true,
-      parallel: true,
+      parallel: !os.release().includes(`Microsoft`),
```

Comment:
Maybe we can use https://github.com/sindresorhus/is-wsl for the check?

Reply:
Thanks for the suggestion. I just added it [right here.](https://github.com/gatsbyjs/gatsby/pull/12636/commits/d29877db36be75d6f1cf73e41b0886d8ed1905a7)